src/Client.ts

```diff
@@ -20,9 +19,11 @@ import { DDiscord, DOption, DSlash } from "./decorators";
 import { GuildNotFoundError } from "./errors";
 
 export class Client extends ClientJS {
+  botid?: string;
+  static importedClass: string[] = []
   private _silent: boolean;
   private _loadClasses: LoadClass[] = [];
-  private static _requiredByDefault: boolean = false;
+  private static _requiredByDefault = false;
   private static _slashGuilds: string[] = [];
   private static _guards: GuardFunction[] = [];
```

Comment:
Without knowing the details, I think these should not be static if you've changed it so there can be multiple instances of a client.

Reply:
already fixed